lib/inc/internal/sources/cpuid_source.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <string>
+
+namespace whereami { namespace sources {
+
+    /**
+     * Register values returned by CPUID
+     */
+    struct cpuid_registers
+    {
+        /**
```

Comment:
If they have to be here, maybe we could make them a little more descriptive? Something like "Value from eax register", etc. Still redundant, but slightly less general (I'd argue any field could accurately be annotated with "<varname> value" :P).